snuba/settings_base.py

```diff
@@ -7,10 +8,13 @@
 
 PORT = 1218
 
+DEFAULT_DATASET_NAME = 'events'
+DISABLED_DATASETS = {}
+LOCAL_DATASET = True
```

Comment:
True means: run everything on one single clickhouse node on a local table (whether in prod or dev)
False means: expect a distributed table.